srcpkgs/emptty/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'emptty'
+pkgname=emptty
+version=0.1.0
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
```

Comment:
`xbps-src` has a `build_style=go` exactly for these types of packages. Trying to build your template as-is leaves some root-owned files in my `masterdir` that cause `./xbps-src clean` to fail. Moving to `build_style=go` causes the binary to be build and installed without these issues. Please switch or fix the file permission problems.

Reply:
If you switch to `build_style=go`, you'll have to set `go_import_path=github.com/tvrzna/emptty`.